### DISCUSSION TITLE
New Sensor keeps stopping

### DISCUSSION THREAD

**Discussion by GLoren1** [Navid200/xDrip#179] [Dexcom]
I changed my sensor. this morning.  Xdrip accepted the code and started the sensor; however a couple of minutes later it stopped. 

The logs show:
Session start failed 1969-16-31 18:00:00

I've tried to restart 3 times with the same error.

**Comment by Navid200** [comment]
Please post in the main repository discussions and I will address it there:
https://github.com/NightscoutFoundation/xDrip/discussions
